src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitedObj = sourceString
+    .split(';')
+    .map((el) => el.trim())
+    .filter((el) => el !== '');
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable. Align the chained methods `.split`, `.map`, and `.filter` vertically for better readability.